tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -82,20 +83,33 @@ def _navigate(self, result: Mapping[str, Any]) -> Any:
         return result
 
     @command
-    def print(
-        self,
-        context: str,
-        background: Optional[bool] = None,
-        margin: Optional[Mapping[str, Any]] = None,
-        orientation: Optional[str] = None,
-        page: Optional[Mapping[str, Any]] = None,
-        page_ranges: Optional[List[str]] = None,
-        scale: Optional[float] = None,
-        shrink_to_fit: Optional[bool] = None
-    ) -> Mapping[str, Any]:
-        params: MutableMapping[str, Any] = {
-            "context": context
-        }
+    def reload(self,
+               context: str,
+               ignore_cache: Optional[bool] = None,
+               wait: Optional[str] = None) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {"context": context}
+        if ignore_cache is not None:
+            params["ignore_cache"] = ignore_cache
+        if wait is not None:
+            params["wait"] = wait
+        return params
+
+    @reload.result
+    def _reload(self, result: Mapping[str, Any]) -> Any:
+        assert result == {}
+        return result
```

Comment:
There is no return value. So this part can be removed to match other commands without results.

Reply:
Thanks! Done